code/modules/mob/living/carbon/human/human.dm

```diff
@@ -334,10 +344,22 @@
 		dat += "<BR><b>Left hand:</b> <A href='?src=\ref[src];item=[slot_l_hand]'>[istype(l_hand) ? l_hand : "nothing"]</A>"
 		dat += "<BR><b>Right hand:</b> <A href='?src=\ref[src];item=[slot_r_hand]'>[istype(r_hand) ? r_hand : "nothing"]</A>"
 
-	// Do they get an option to set internals?
-	if(istype(wear_mask, /obj/item/clothing/mask) || istype(head, /obj/item/clothing/head/helmet/space))
-		if(istype(back, /obj/item/tank) || istype(belt, /obj/item/tank) || istype(s_store, /obj/item/tank))
-			dat += "<BR><A href='?src=\ref[src];item=internals'>Toggle internals.</A>"
+	var/has_mask // 0, no mask | 1, mask but it's down | 2, mask and it's ready
```

Comment:
Why do you need these specific values assigned to this? As far as I can tell, they're not actually used.

Reply:
honestly it seems like future-proofing; line 361 uses it, i could see adding an "adjust mask" option or something there in the future

Reply:
hah nevermind there actually is that option there! what do you know